src/main/kotlin/com/blincke/commune_api/models/database/users/User.kt

```diff
@@ -20,53 +20,16 @@ class User(
 
     @Column(name = "profile_picture_url")
     val profilePictureUrl: String? = null,
-
-    @OneToMany(mappedBy = "friendshipId.requester", cascade = [CascadeType.ALL], orphanRemoval = true)
```

Comment:
This is where I removed the big lists. They won't usually be a problem because execution is LAZY, but I think removing them removes the possibility of shooting ourselves in the foot.